app/gateway/resource-sizing-guidelines.md

```diff
@@ -0,0 +1,178 @@
+CPU available for processing traffic as a whole. {{site.base_gateway}} is
+designed to scale horizontally to add more overall compute power for
+configurations that add substantial latency into requests, while needing to
+meet specific throughput requirements.
+
+Performance benchmarking and optimization as a whole is a complex exercise that
+must account for a variety of factors, including those external to
+{{site.base_gateway}}, such as the behavior of upstream services, or the health
+of the underlying hardware on which {{site.base_gateway}} is running.
+
+## General resource guidelines
+
+These recommendations are a baseline guide only. 
+For performance-critical environments, you should conduct specific [tuning or benchmarking efforts](/gateway/performance/establish-performance-benchmark/).
+
+{% if_version gte:3.5.x %}
+### Hybrid mode with large number of entities
+
+When {{site.base_gateway}} is operating in hybrid mode with a large number of
+[entities](/gateway/entities/) (like Routes and Gateway Services), it can benefit from enabling [`dedicated_config_processing`](/gateway/configuration/#dedicated_config_processing).
+
+When enabled, certain CPU-intensive steps of the data plane reconfiguration operation are offloaded
+to a dedicated worker process. This reduces proxy latency during reconfigurations at the cost of a
+slight increase in memory usage. The benefits of this are most apparent with configurations
+of more than 1,000 entities. 
+{% endif_version %}
+
+### {{site.base_gateway}} resources
+
+{{site.base_gateway}} is designed to operate in a variety of deployment
+environments. It has no minimum system requirements to operate.
+
+Resource requirements vary substantially based on configuration. The following
+high-level matrices offer a guideline for determining system requirements
+based on overall configuration and performance requirements.
+
+The following table provides rough usage requirement estimates based on simplified examples with latency and throughput requirements on a per-node basis:
+
+| Size | Number of configured entities | Latency requirements | Throughput requirements | Use cases |
+|---|---|---|---|---|
+| Development | < 100 | < 100 ms | < 500 RPS   | * Dev/test environments<br>* Latency-insensitive gateways |
+| Small | < 1000  | < 20 ms  | < 2500 RPS  | * Production clusters<br>* Greenfield traffic deployments |
+| Medium | < 10000 | < 10 ms  | < 10000 RPS | * Mission-critical clusters<br>* Legacy and greenfield traffic<br>* Central enterprise-grade gateways |
+| Large | < 50000+ | < 10 ms  | < 10000 RPS | * Mission-critical clusters<br>* Legacy and greenfield traffic<br>* Central enterprise-grade gateways |
+
+### Database resources
+
+We do not provide any specific numbers for database sizing because it
+depends on your particular setup. Sizing varies based on:
+* Traffic
+* Number of nodes
+* Enabled features
+  *For example: [Rate limiting](/gateway/rate-limiting/) uses a database or Redis*
```

Comment:
```suggestion
* Enabled features
  
  *For example: [Rate limiting](/gateway/rate-limiting/) uses a database or Redis*
```
If you want a linebreak here. Right now it displays like this: 
<img width="545" alt="Screenshot 2025-02-28 at 2 03 55 PM" src="https://github.com/user-attachments/assets/b5539f51-94a5-4635-8df8-16ee28545a95" />